apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/schemas/Sgv2ColumnsResourceApi.java

```diff
@@ -54,7 +55,7 @@ public interface Sgv2ColumnsResourceApi {
         @APIResponse(ref = RestOpenApiConstants.Responses.GENERAL_404),
         @APIResponse(ref = RestOpenApiConstants.Responses.GENERAL_500)
       })
-  Response getAllColumns(
+  Uni<RestResponse<Object>> getAllColumns(
```

Comment:
Ok; I followed example of Docs API I think -- but that's probably not applicable. Will create a follow-up issue.

Reply:
Yea you should never directly return `ApiError`, but let mappers do transformation from a failure..

Reply:
@ivansenic I started working on this and can improve usage in many places. But one common problematic pattern is the "raw" handling: basically having either "ResponseWrapper<ResultValue>" OR "ResultValue". Since Java doesn't really have union types it's difficult to model it.

So I will add types where applicable (there are some `Map<String,String>`s and a few `Void`s (for DELETEs)), and then registration annotations where common base type not available.